client/src/pages/Jessica.jsx

```diff
@@ -1,13 +1,57 @@
 // @flow
 import React from 'react';
 
-import '../css/Home.css';
+import '../css/Jessica.css';
+import bg from '../utils/images/bg.jpg';
+import sushi from '../utils/images/sushi.jpg';
+import mic from '../utils/images/mic.PNG';
+import bm from '../utils/images/badminton.PNG';
 
 function Jessica() {
   return (
-    <>
-      <h1>Jessica's Page</h1>
-    </>
+    <div className="main">
+      <img src={bg} alt="Mount Rainier" width="100%" height="400"></img>
+
+      <div className="name"> Jessica Singh </div>
+      <h2>H4i Academy Member and Freshman in CS</h2>
+      <div className="container">
+        <div className="image">
```

Comment:
would recommend more specific classNames like "jessica-content-container" and "sushi-img-container" to avoid styling conflicts (in case another member has a className "container" or "image")